executor/README.md

```diff
@@ -0,0 +1,270 @@
+Cook Executor
+=============
+
+The cook executor is a custom executor written in python. It replaces the default command executor in order to enable a number of features for both operators and end users.
```

Comment:
Out of curiosity, why was Python chosen vs other languages such as Clojure/Java?